feature-utils/silly-i18n/rollup-plugin.cjs

```diff
@@ -0,0 +1,57 @@
+const fs = require("fs");
+const path = require("path");
+const util = require("util");
+const glob = require("glob");
+
+const DIRNAME = "locales";
+const FILEEXT = ".json";
+const PLUGIN = "silly-i18n";
+const SUFFIX = `?${PLUGIN}`;
+
+exports.default = function (options) {
+    return {
+        name: PLUGIN,
+
+        async resolveId(source, importer) {
+            if (source == `!${PLUGIN}`) {
+                for (let dir = importer; dir != (dir = path.dirname(dir)); ) {
+                    const subdir = path.join(dir, DIRNAME);
+                    const exists = await fs.promises.stat(subdir).then(
+                        (stat) => stat.isDirectory(),
+                        () => false
+                    );
+                    if (exists) return `${subdir}${SUFFIX}`;
```

Comment:
Where is this `exists` which is defined above uses its definition? Or is it using just the fact that it's truish?

Reply:
`exists` is boolean that is `true` if a directory with the name "locales" exists within the current directory, otherwise `false`. What do you mean?